server/routes/record.js

```diff
@@ -52,6 +52,7 @@ recordRoutes.route("/record/add").post(function (req, response) {
    sex: req.body.sex,
    height: req.body.height,
    weight: req.body.weight,
+   bsa: Math.sqrt((+req.body.height * +req.body.weight) / 3600),
```

Comment:
Don't forget to also include `bsa` as an attribute through the `update/:id` route after line 75.